### DISCUSSION TITLE
cesium/widgets.css文件加载不到

### DISCUSSION THREAD

**Discussion by sunny-0512** [nshen/vite-plugin-cesium#25] [Q&A]
import 'cesium/widgets.css' // 加载不到

在博客https://www.cnblogs.com/zfygiser/p/15239818.html上找到一个解决方案是修改node_modules中依赖源文件package.json，在exports输出中，添加widgets文件的路径输出。

这种方式确实能解决问题，但不够简洁。
希望得到您的指点。

**Comment by nshen** [comment]
难道是cesium版本更新了的问题？
